packages/v4/patternfly-docs/content/design-guidelines/styles/colors/colors.md

```diff
@@ -33,16 +33,16 @@ import './colors.css';
   <GridItem xl={6} lg={12} className="ws-colors-gridtext">
     ### Text and icons
     <p>
-      Typography and iconography colors are extremely similar because they can both be used in the same context, such as a link button. We have multiple shades of text colors on light and dark backgrounds so that you can emphasize hierarchy.
+      Typography and iconography colors are extremely similar because they can be used in the same context, such as for a link button. We have multiple shades of text colors on light and dark backgrounds so that you can emphasize hierarchy.
     </p>
     <ColorSwatch color="--pf-global--Color--100">
-      This color is used as a standard text and icon color as well as a hover state color for icon buttons. It is most commonly used as a text color for many components and application screens.
+      This color is used for standard text, icons, and as a hover state color for icon buttons. It is most commonly used as a text color for many components and application screens.
     </ColorSwatch>
     <ColorSwatch color="--pf-global--Color--200">
-      This color is used as a secondary text color as well as a default color for icon buttons. It is most commonly used for many components, such as the app launcher and empty states.
+      This color is used as a secondary text color and as a default color for icon buttons. It is most commonly used for many components, such as the app launcher and empty states.
     </ColorSwatch>
     <ColorSwatch color="--pf-global--link--Color">
-      This color is most commonly used as a link text and icon color for many components, such as navigation and accordions. Blue icons use the same primary color variable as the blue text.
+      This color is most commonly used for link text and icons for many components, such as navigation and accordions. Blue icons use the same primary color variable as the blue text.
```

Comment:
> Blue icons use the same primary color variable as the blue text.

If this is referring to the link button, I think "primary color variable" is confusing, since "primary-color--[number]" is a variable and is different. Assuming it's the link button, I'd just say something like icons that are part of links use the same (link?) color variable as the text.